frontend/src/pages/org/workflow-editor.ts

```diff
@@ -666,16 +667,11 @@ export class CrawlConfigEditor extends LiteElement {
           <btrix-tab-panel name="newJobConfig-crawlSetup" class="scroll-m-3">
             ${this.renderPanelContent(
               html`
-                ${when(this.jobType === "url-list", () =>
```

Comment:
Interesting, I don't see this documented in the lit v3 docs but the type indicates that `when1 now passes the condition to the callbacks. Probably easier to read this logic with `choose`, which I updated with this commit?